tools/update_tool

```diff
@@ -0,0 +1,466 @@
+#!/bin/env python
+
+import argparse
+import json
+import os
+import sys
+import tempfile
+import shlex
+import subprocess
+import columnify
+
+RESET = "\033[0m"
+GREEN = "\033[32m"
+BOLD = "\033[1m"
+RED = "\033[31m"
+YELLOW = "\033[33m"
+STRIKE = "\033[9m"
+
+# Utility functions to print an ascii table and a github markdown lits
+
+def start_row(text, gh):
+    end = "|"
+    prefix = ""
+    l = 77
+    if gh:
+        l = 0
+        end = ", "
+        prefix = "- [ ] "
+
+    print(f"{prefix}{text}".ljust(l), end=end)
+
+
+def green_adjusted(text, bold=False, strike=False, l=18, gh=False, last=False):
```

Comment:
`green_adjusted` is basically the same thing as `red_adjusted`. Btw, have you considered using something like `jinja` to do the formatting? I think a declarative template might be simpler that doing the formatting imperatively,

Reply:
Yes, that chunk of code is not my favourite here. You can see that I went back and forth many times and that are leftovers from the first ideas. That needs a cleanup.

Reply:
I am irked by the formatting code as well, I wouldn't suggest going down the actual templating engine path but I *would* suggest moving all the text formatting functions into a class, having the class encapsulate the `is_github` boolean.

Most of these formatting functions can also be 'cleaned up' but that's just an opinion; e.g. I'd write them more in this idiom:

```
def start_row(text, is_github):
    min_width = 77

    end = ", " if is_github else "|"
    pre = "- [ ] " if is_github else ""

    print(f"{pre}{text:<{min_width}}{end}", end="")
```

You can then move shared bits into the format class (e.g; `end = "|"`) is repeated a few times, so are others.